doozer

```diff
@@ -1454,6 +1454,36 @@ def config_print(runtime, key, name_only, as_yaml):
     config.config_print(key, name_only, as_yaml)
 
 
+@cli.command("config:gen-csv", short_help="Generate .csv file for given images/rpms")
+@click.option("--fields", help="Specific key in config to print, separated by commas: --fields key,name,owners",
+              default=None)
+@click.option("--type", "as_type", default=None, help='Write content type: image or rpm')
```

Comment:
Need to add `--output / -o` option to specify output file name and directory. Without reading the code it's not obvious where this outputs to. If `--output` is not specified, just emit to `stdout`.